shared/featureconfig/config.go

```diff
@@ -114,12 +116,21 @@ func InitWithReset(c *Flags) func() {
 	return resetFunc
 }
 
+// VerifyTestnet checks for required testnet flag
+func VerifyTestnet(ctx *cli.Context) error {
+	if !ctx.Bool(AltonaTestnet.Name) &&
+		!ctx.Bool(OnyxTestnet.Name) &&
+		!ctx.Bool(MedallaTestnet.Name) &&
+		!ctx.Bool(SpadinaTestnet.Name) &&
+		!ctx.Bool(ZinkenTestnet.Name) {
```

Comment:
Please add `E2EConfigFlag`. Otherwise these constraints break e2e testing.

https://github.com/prysmaticlabs/prysm/blob/002bbf53d0cf27e17d455581b968b2a52deb2810/shared/cmd/flags.go#L15